### PR TITLE
Replace Hyperspace with Calibration placeholders

### PR DESCRIPTION
**Pull request by avichalp**
Fixes EVM-49

### CODE REVIEW

**Comment by sanderpick** [comment]
looks good! I could do the deploy and update this contract is that works

**Comment by sanderpick** [comment]
deployed and updated

File: hardhat.config.ts [hunk 1]
```diff
@@ -167,13 +167,13 @@ const config: HardhatUserConfig = {
           ? [process.env.POLYGON_MUMBAI_PRIVATE_KEY]
           : [],
     },
-    "filecoin-hyperspace": {
-      url: `https://rpc.ankr.com/filecoin_testnet/${
-        process.env.FILECOIN_HYPERSPACE_API_KEY ?? ""
+    "filecoin-calibration": {
+      url: `https://api.calibration.node.glif.io/rpc/v1${
```
Comment (avichalp) [inline hardhat.config.ts:171]: using Glif.io instead of Ankr because they don't support the calibration network yet

**Review by joewagner — APPROVED** [review]
This looks good to me!  
Will the changes to the Validator be ready to release the coming week?

**Comment by avichalp** [comment]
@joewagner: yes, I think so. gonna make the validator PR tomorrow

**Pull request merged**